tasks/build_tags.py

```diff
@@ -316,7 +317,7 @@ def filter_incompatible_tags(include, platform=sys.platform):
     if not platform.startswith("linux"):
         exclude = exclude.union(LINUX_ONLY_TAGS)
 
-    if platform == "win32":
+    if platform == "win32" or os.getenv("GOOS") == "windows":
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities. -->
**💭 thought**
Wonder if we don't have a `is_windows` helper somewhere

Reply:
We do but it means "are we running Windows" more than "are we targeting Windows"